recipe/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+$PYTHON setup.py install --conda
+
+cp ${RECIPE_DIR}/xonsh_shortcut.json ${PREFIX}/Menu/xonsh_shortcut.json
+cp ${RECIPE_DIR}/xonsh.ico ${PREFIX}/Menu/xonsh.ico
```

Comment:
Good point. I am not sure how it look on a linux anaconda installation. I got the idea from the spyder packages. I will check how it looks on for the corresponding linux packages.